Library/ViewModels/ManagePledgeSummaryViewViewModel.swift

```diff
@@ -0,0 +1,136 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveSwift
+
+public protocol ManagePledgeSummaryViewViewModelInputs {
+  func configureWith(_ project: Project)
+}
+
+public protocol ManagePledgeSummaryViewViewModelOutputs {
+  var backerNumberText: Signal<String, Never> { get }
+  var backingDateText: Signal<String, Never> { get }
+  var pledgeAmountText: Signal<NSAttributedString, Never> { get }
+  var shippingAmountText: Signal<NSAttributedString, Never> { get }
+  var shippingLocationStackViewIsHidden: Signal<Bool, Never> { get }
+  var shippingLocationText: Signal<String, Never> { get }
+  var totalAmountText: Signal<NSAttributedString, Never> { get }
+}
+
+public protocol ManagePledgeSummaryViewViewModelType {
+  var inputs: ManagePledgeSummaryViewViewModelInputs { get }
+  var outputs: ManagePledgeSummaryViewViewModelOutputs { get }
+}
+
+public class ManagePledgeSummaryViewViewModel: ManagePledgeSummaryViewViewModelType,
```

Comment:
I'm not sure when we started calling VM's VVM's 😅. I don't think we should be naming them `ViewViewModel`, the fact that this VM is used for a View and not a ViewController doesn't matter, we should just call it `ManagePledgeSummaryViewModel`, I think we should also add an investment day task to fix these names in the codebase for other VVMs.

Reply:
lol it makes sense. I will rename it